ansible/roles/kong-api/defaults/main.yml

```diff
@@ -1553,6 +1571,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: privateListUserCourseEnrollments
+    uris: "{{ course_service_prefix }}/private/v2/user/enrollment/list"
+    upstream_url: "{{ lms_service_url }}/private/v2/user/courses/list"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'anonymousCourseAccess'
```

Comment:
So this API can be called without logging in or X-Auth-Token?

Reply:
Yes, these are private APIs for internal purpose

Reply:
As discussed, I have added the new ACL and updated APIs list also